readiness.md

```diff
@@ -0,0 +1,191 @@
+aware that they need open source capabilities.  The hard part is
+knowing where to start in adding skills.  In most cases, the most
+important consideration is building capability to know how and when to
+use FOSS strategies.
+
+Open source is all about execution.  Reaping the benefits of open
+source investment requires completing a series of difficult steps
+ranging from designing an initial strategy to building an appropriate
+community to leveraging the resulting dynamics for strategic
+advantage.  Most government procurement teams are not yet prepared to
+travel that path and reap those rewards at scale.  They are still
+growing in their ability to execute.  As an agency starts to climb the
+learning curve, it can be helpful to assess current abilities as a
+form of organizational readiness, and to describe the journey to
+mastery as one of gaining capability.
+
+To do that, we might locate current capabilities in a readiness model.
+This clarifies where an agency is in its journey to mastery and also
+suggests next steps and likely results.  Often, teams use these models
+to identify areas for potential growth.  They are most useful for
+those early in their open source progress, and this module focuses on
+those beginning stages.  It is useful for an agency to consider its
+maturity on an organization wide basis, but open source is usually
+executed by teams, and it is the capabilities of specific teams that
+matter most.  This module considers individual teams as the target for
+analysis because even agency-wide open source skills training will
+succeed or fail mostly at the level of specific teams working on
+specific projects.
+
+There are several readiness models that go by various names.
+Practitioners often describe climbing the "readiness ladder."
+
+ * [TODO Group model](https://github.com/todogroup/ospo101/blob/main/module2/README.md#climbing-the-open-source-ladder)
+
+ * [Jeff McAffer's Engagement Model](https://mcaffer.com/2019/02/Open-source-engagement)
+
+ * [GitHub's Maturity Model](https://github.com/github/maturity-model)
+
+ * [Qalipso's Model](https://en.wikipedia.org/wiki/OpenSource_Maturity_Model)
+
+ * [Stephen Walli's Maturation Toward Adulthood](https://opensource.com/life/15/4/what-does-an-adult-look-like-in-an-open-source-community)
+
+ * [FinOS Open Source Ladder](https://www.finos.org/hubfs/FINOS/website/pages/community/Open%20Source%20Readiness%20Roadmap%20-%203.10.2018.png)
+
+As you can see, there are a number of such models.  It is often worth
+considering more than one when examining an organization or a
+particular team.  Of the published models, we find McAffer's in
+particular is useful because it includes strategic components.  It
+accounts for realistic failure modes, and understands that open source
+readiness will be unevenly distributed in any agency large enough to
+have multiple teams.
+
+
+FURTHER READING Microsoft Senior Director Jeff McAffer (now Senior Director at GitHub) has published a
```

Comment:
This reads strange to me in that you are linking twice to the same resource, though I understand that you are highlighting one of the five resources that you link above, but you may consider alternative wording, as in the way it is written it almost reads as yet another resource in addition to the ones listed above